theseus/geometry/functional/lie_group.py

```diff
@@ -27,16 +27,24 @@ def UnaryOperatorFactory(module, op_name):
     op_autograd_fn = getattr(module, "_" + op_name + "_autograd_fn")
     jop_autograd_fn = getattr(module, "_j" + op_name + "_autograd_fn")
 
-    def op(
-        input: torch.Tensor,
-        jacobians: Optional[List[torch.Tensor]] = None,
-    ) -> torch.Tensor:
-        if jacobians is not None:
-            check_jacobians_list(jacobians)
-            jacobians.append(jop_autograd_fn(input)[0])
-        return op_autograd_fn(input)
-
-    def jop(input: torch.Tensor) -> Tuple[List[torch.Tensor], torch.Tensor]:
-        return jop_autograd_fn(input)
-
-    return op, jop
+    if jop_autograd_fn is not None:
+
+        def op(
+            input: torch.Tensor,
+            jacobians: Optional[List[torch.Tensor]] = None,
+        ) -> torch.Tensor:
+            if jacobians is not None:
+                check_jacobians_list(jacobians)
+                jacobians.append(jop_autograd_fn(input)[0])
+            return op_autograd_fn(input)
+
+        def jop(input: torch.Tensor) -> Tuple[List[torch.Tensor], torch.Tensor]:
+            return jop_autograd_fn(input)
+
+        return op, jop
+    else:
+
+        def op_no_jop(input: torch.Tensor) -> torch.Tensor:
+            return op_autograd_fn(input)
+
+        return op_no_jop
```

Comment:
I am guessing this block will be removed once we have added `_jxxx_autograd_fn`?

Reply:
No, the jacobian of `adjoint` transformation has no meaning and is not well defined. Despite that, we still provide analytic backward.

Reply:
Ah okay, it would useful to add this in comments so it's clear it's not something we will be adding later like the autograd for some other jacobians.